docs/releases/5.0.txt

```diff
@@ -282,6 +282,12 @@ Internationalization
 
 * ...
 
+Initialization process
+~~~~~~~~~~~~~~~~~~~~~~
+
+* :func:`django.setup` raises a ``RuntimeWarning`` when apps interact with
+  the database before the app registry has been fully populated.
```

Comment:
I just put it there because I usually run with `-Werror` during development, so this would kind of be a breaking change for me. But removed nonetheless.

Reply:
If you want to treat this as a backward incompatible change, than note a should be added to the `Backwards incompatible changes in 5.0` -> `Miscellaneous`.